engine/src/exec-interface-image.cpp

```diff
@@ -313,46 +318,75 @@ void MCImage::GetFormattedWidth(MCExecContext& ctxt, integer_t& r_width)
 
 void MCImage::GetText(MCExecContext& ctxt, MCDataRef& r_text)
 {
-	recompress();
-	if (m_rep == nil || m_rep->GetType() == kMCImageRepReferenced)
-	{
-		r_text = MCValueRetain(kMCEmptyData);
-		return;
-	}
-	else
+    
+    
+    if (m_rep == nullptr || m_rep->GetType() == kMCImageRepReferenced)
+    {
+        r_text = MCValueRetain(kMCEmptyData);
+        return;
+    }
+    
+    bool t_success = false;
+    
+    
+    MCImage* t_image_to_work_on = this;
+    
```

Comment:
This var definition is setup by the subsequent clause, so keep it together (i.e. remove the newline after).